src/pages/workspaces/List.js

```diff
@@ -95,15 +95,15 @@ const workspaceSubmissionStatus = workspace => {
 }
 
 // [PPWM-104] Temporary logic supporting project-per-workspace persistent disk migration
-const anyPersistentDiskRequiresMigration = (workspaces, disks) => {
+const somePersistentDiskRequiresMigration = (workspaces, disks) => {
   // construct map of namespace -> (map of name -> workspace) from list of workspace
   const workspacesByNsAndName = _.flow(
     _.map('workspace'),
     _.groupBy('namespace'),
     _.mapValues(_.flow(_.groupBy('name'), _.mapValues(_.head)))
   )(workspaces)
 
-  return _.flip(_.some)(disks, ({ googleProject, labels }) => {
+  return workspaces && _.flip(_.some)(disks, ({ googleProject, labels }) => {
```

Comment:
So this works OK if `disks` is undefined?

Reply:
yes  - you can verify this yourself in the console: 
```javascript
> _.some(() => true, undefined)
false
```